SAMPLE_FORMAT.md

```diff
@@ -1,15 +1,47 @@
 # Samples Format
+## Table of Contents
+  * [Java Version](#java-version)
+  * [Specific Goals](#specific-goals)
+  * [Testing](#testing)
+    * [Testing Setup](#testing-setup)
+    * [Running Locally](#running-locally)
+    * [Gradle Specifics](#gradle-specifics)
+  * [Format Guidelines](#format-guidelines)
+    * [Location](#project-location)
+    * [Dependencies](#project-dependencies)
+    * [Configuration](#project-configuration)
+    * [Setup](#project-setup)
+  * [Code](#code)
+    * [Arrange, Act, Assert](#arrange-act-assert)
+    * [Style](#style)
+    * [Linting](#linting)
+    * [Command-Line Arguments](#command-line-arguments)
+    * [Package Names](#package-names)
+    * [Class Structure]()
+    * [Function Comment]()
+    * [Function Structure]()
+    * [Exception Handling]()
+    * [Client Initialization](#client-initialization)
+  * [Modern Java](#modern-java)
+    * [Lambdas](#lambdas)
+    * [Streams](#streams)
+    * [Parallel Streams](#parallel-streams)
+  * [Additional Best Practices](#additional-best-practices)
+    * [Time](#time)
+    * [Logging](#logging)
 
 This doc maintains an outline for 'snippet' samples specific to Java. Currently, the java canonical
 samples in this format are located 
-[here](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/dlp/src/main/java/dlp/snippets).
+[here](/tree/master/dlp/src/main/java/dlp/snippets).
```

Comment:
Pretty sure this is just cus this is on a branch, but the link doesn't work.

Reply:
If there is a problem, I'll fix it w/ trivial PR.